include/SDL3/SDL_tray.h

```diff
@@ -0,0 +1,486 @@
+
+/**
+ * Returns a list of entries in the menu, in order.
+ *
+ * \param menu The menu to get entries from.
+ * \param size An optional pointer to obtain the number of entries in the menu.
+ * \returns the entries within the given menu. The pointer becomes invalid when
+ *          any function that inserts or deletes entries in the menu is called.
+ *
+ * \since This function is available since SDL 3.0.0.
+ *
+ * \sa SDL_GetTrayEntryAt
+ * \sa SDL_RemoveTrayEntry
+ * \sa SDL_RemoveTrayEntryAt
+ * \sa SDL_InsertTrayEntryAt
+ * \sa SDL_AppendTrayEntry
+ */
+extern SDL_DECLSPEC const SDL_TrayEntry **SDLCALL SDL_GetTrayEntries(SDL_TrayMenu *menu, int *size);
+
+/**
+ * Returns a tray entry at a given position.
+ *
+ * \param menu The menu to get the entry from.
+ * \param pos the position of the desired entry.
+ * \returns the entry, of NULL if pos is out of bounds.
+ *
+ * \since This function is available since SDL 3.0.0.
+ *
+ * \sa SDL_GetTrayEntries
+ * \sa SDL_RemoveTrayEntry
+ * \sa SDL_RemoveTrayEntryAt
+ * \sa SDL_InsertTrayEntryAt
+ * \sa SDL_AppendTrayEntry
+ */
+extern SDL_DECLSPEC SDL_TrayEntry *SDLCALL SDL_GetTrayEntryAt(SDL_TrayMenu *menu, int pos);
+
+/**
+ * Removes a tray entry.
+ *
+ * \param entry The entry to be deleted.
+ *
+ * \since This function is available since SDL 3.0.0.
+ *
+ * \sa SDL_GetTrayEntries
+ * \sa SDL_GetTrayEntryAt
+ * \sa SDL_RemoveTrayEntryAt
+ * \sa SDL_InsertTrayEntryAt
+ * \sa SDL_AppendTrayEntry
+ */
+extern SDL_DECLSPEC void SDLCALL SDL_RemoveTrayEntry(SDL_TrayEntry *entry);
+
+/**
+ * Removes a tray entry at a given position.
+ *
+ * \param menu The menu to get the entry from.
+ * \param pos the position of the entry to be deleted.
+ *
+ * \since This function is available since SDL 3.0.0.
+ *
+ * \sa SDL_GetTrayEntries
+ * \sa SDL_GetTrayEntryAt
+ * \sa SDL_RemoveTrayEntry
+ * \sa SDL_InsertTrayEntryAt
+ * \sa SDL_AppendTrayEntry
+ */
+extern SDL_DECLSPEC void SDLCALL SDL_RemoveTrayEntryAt(SDL_TrayMenu *menu, int pos);
+
+/**
+ * Insert a tray entry at a given position.
+ *
+ * An entry does not need to be destroyed; it will be destroyed with the tray.
+ *
+ * \param menu the menu to append the entry to.
+ * \param pos the desired position for the new entry. Entries at or following
+ *            this place will be moved.
+ * \param label the label to be displayed on the entry.
+ * \param flags a combination of flags, some of which are mandatory.
+ * \returns the newly created entry, or NULL if pos is out of bounds.
+ *
+ * \since This function is available since SDL 3.0.0.
+ *
+ * \sa SDL_GetTrayEntries
+ * \sa SDL_GetTrayEntryAt
+ * \sa SDL_RemoveTrayEntry
+ * \sa SDL_RemoveTrayEntryAt
+ * \sa SDL_AppendTrayEntry
+ * \sa SDL_TrayEntryFlags
+ * \sa SDL_CreateTrayMenu
+ * \sa SDL_CreateTraySubmenu
+ * \sa SDL_SetTrayEntryChecked
+ * \sa SDL_GetTrayEntryChecked
+ * \sa SDL_SetTrayEntryEnabled
+ * \sa SDL_GetTrayEntryEnabled
+ * \sa SDL_SetTrayEntryCallback
+ * \sa SDL_AppendTraySeparator
```

Comment:
You don't need to include all the related functions, go ahead and reduce it to the function that creates the object you're working with and functions directly related to this one.